lua/lazyvim/plugins/extras/editor/mini-files.lua

```diff
@@ -26,4 +26,24 @@ return {
       desc = "Open mini.files (cwd)",
     },
   },
+  init = function()
```

Comment:
`init` is not ideal for this, since this will always be loaded.

Can you change this to the config function below?

```lua
  config = function(_, opts)
    require("mini.files").setup(opts)
    -- add that toggle code here 
  end,
```

Reply:
Done. I found that the setup call is not needed, but added anyway.